app/controllers/api/emoji_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::EmojiController < ApplicationController
```

Comment:
__[rubocop]__
* Missing top-level class documentation comment.
* Use nested module/class definitions instead of compact style.